setup.py

```diff
@@ -5,6 +5,10 @@
 #   DEBUG
 #     build with -O0 and -g (debug symbols)
 #
+#   VERBOSE
+#     run ninja/make verbosely, printing out each compile command as it
+#     compiles commands
+#
```

Comment:
nit: I would put this below REL_WITH_DEB_INFO

Reply:
sure :>